examples/widgets/bottom_sheet.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
I think we're better off adding this kind of demo to an existing demo rather than creating a demo for each feature. (Do create a test for each feature, though!)

The problem with the demos is we hardly ever run them so they bit rot quite fast.